package.json

```diff
@@ -1,56 +1,59 @@
-    "cryptiles": "^4.1.3",
-    "debug": "~4.1.1",
-    "dotenv": "^8.2.0",
-    "express": "~4.17.1",
-    "express-validator": "^6.3.0",
-    "express-winston": "^2.6.0",
-    "handlebars": "^4.5.3",
-    "jsonwebtoken": "^8.5.1",
-    "memory-cache": "^0.2.0",
-    "mongoose": "^5.7.14",
-    "multer": "^1.4.2",
-    "passport": "^0.4.0",
-    "passport-local": "^1.0.0",
-    "q": "^1.5.1",
-    "qrcode": "^1.4.4",
-    "winston": "^2.4.4"
-  },
-  "devDependencies": {
-    "@types/express": "^4.17.2",
-    "@types/google-cloud__storage": "^1.7.2",
-    "@types/mongodb": "^3.3.12",
-    "@types/mongoose": "^5.5.32",
-    "@types/multer": "^1.3.10",
-    "apidoc": "^0.17.7",
-    "chai": "^4.2.0",
-    "chai-http": "^4.3.0",
-    "eslint": "6.7.2",
-    "eslint-config-prettier": "6.7.0",
-    "eslint-plugin-prettier": "3.1.1",
-    "mocha": "^6.2.2",
-    "nodemon": "^1.19.4",
-    "prettier": "1.19.1"
-  }
+    "name": "hackerAPI",
+    "version": "1.6.2",
+    "private": true,
+    "scripts": {
+        "start": "DEBUG=hackboard:* NODE_ENV=development nodemon --ignore gcp_creds.json ./bin/www.js",
+        "deploy": "NODE_ENV=deployment node ./bin/www.js",
+        "debug": "DEBUG=hackboard:* NODE_ENV=deployment nodemon --ignore gcp_creds.json ./bin/www.js",
+        "test": "DEBUG=hackboard:* NODE_ENV=test mocha --reporter spec tests/**.js --exit",
+        "seed": "NODE_ENV=development node ./seed/index.js",
+        "docs": "apidoc -i ./routes -o ./docs/api/",
+        "format": "prettier --write '**/*.js'",
+        "lint": "eslint --fix '**/*.js'"
+    },
+    "engines": {
+        "node": "10.17.0"
+    },
+    "dependencies": {
+        "@google-cloud/logging-winston": "^0.11.1",
+        "@google-cloud/storage": "^3.5.0",
+        "@sendgrid/mail": "^6.4.0",
+        "bcrypt": "^3.0.7",
+        "cookie-parser": "~1.4.4",
+        "cookie-session": "^2.0.0-beta.3",
+        "cors": "^2.8.5",
+        "cryptiles": "^4.1.3",
+        "debug": "~4.1.1",
+        "dotenv": "^8.2.0",
+        "express": "~4.17.1",
+        "express-validator": "^6.3.0",
+        "express-winston": "^2.6.0",
+        "handlebars": "^4.5.3",
+        "jsonwebtoken": "^8.5.1",
+        "memory-cache": "^0.2.0",
+        "mongoose": "^5.7.14",
+        "multer": "^1.4.2",
+        "passport": "^0.4.0",
+        "passport-local": "^1.0.0",
+        "q": "^1.5.1",
+        "qrcode": "^1.4.4",
+        "winston": "^2.4.4"
+    },
+    "devDependencies": {
+        "@types/express": "^4.17.2",
+        "@types/google-cloud__storage": "^1.7.2",
+        "@types/mongodb": "^3.3.12",
+        "@types/mongoose": "^5.5.32",
+        "@types/multer": "^1.3.10",
+        "apidoc": "^0.17.7",
+        "chai": "^4.2.0",
+        "chai-http": "^4.3.0",
+        "eslint": "6.7.2",
+        "eslint-config-prettier": "6.7.0",
+        "eslint-plugin-prettier": "3.1.1",
+        "mocha": "^6.2.2",
+        "nodemon": "^1.19.4",
+        "prettier": "1.19.1"
+    }
```

Comment:
The VS Code format on save settings forced the 4 tab spacing to be applied to this file hence the large diff.